src/ssl.mli

```diff
@@ -200,9 +200,26 @@ exception Verify_error of verify_error
     [Ssl_threads.init] first. *)
 val init : ?thread_safe:bool -> unit -> unit
 
-(** Retrieve a human-readable message that corresponds to the last error that
-    occurred. *)
+(** Retrieve a human-readable message that corresponds to the earliest error
+    code from the thread's error queue and removes the entry. *)
 val get_error_string : unit -> string
+  [@@ocaml.alert deprecated "Use [Ssl.Error.get_error] instead"]
+
+module Error: sig
```

Comment:
@Firgeis In a follow-up PR, it'd be nice to explain in this doc that OpenSSL keeps an error queue per thread, that it gets cleaned up before every TLS operation, which can then lead to users of ocaml-ssl understanding what it means to get vs peek errors.

Reply:
@anmonteiro  When you say in this doc, you mean as a comment for the error module?

Reply:
yeah, in this `.mli` file.